### PR TITLE
feat: add flow id to meta

### PR DESCRIPTION
**Pull request by IyadhKhalfallah**
---

- [ ] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

File: now/executor/gateway/hubble_report.py [hunk 1]
```diff
@@ -99,6 +100,7 @@ def report(quantity_basic, quantity_pro, auth_jwt=None):
                     'appId': 'search-app',
                     'feeType': 'search-fee',
                     'source': 'gateway',
+                    'flowId': namespace.split('-')[1] if namespace else namespace,
```
Comment (makram93) [inline now/executor/gateway/hubble_report.py:103]: you can rely on `user_input.flow_name`
```suggestion
                    'flowId': user_input.flow_name + '-' + namespace,
```

**Review by makram93 — CHANGES_REQUESTED** [review]

**Comment by shazhou2015** [comment]
Is this PR ready to merge now? @IyadhKhalfallah @makram93

File: now/executor/gateway/hubble_report.py [hunk 2]
```diff
@@ -85,6 +85,7 @@ def report(quantity_basic, quantity_pro, auth_jwt=None):
         summary = get_summary(auth_jwt=authorized_jwt)
         product_id = summary['internal_product_id']
         logger.info(f'Credits before: {summary["credits"]} & product_id: {product_id}')
+        namespace = os.environ.get('K8S_NAMESPACE_NAME')
```
Comment (makram93) [inline now/executor/gateway/hubble_report.py:88]: We can remove this, now that we have full flow name

**Comment by girishc13** [comment]
LGTM

**Review by Tanguyabel — APPROVED** [review]

**Pull request merged**